modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractGoCodegen.java

```diff
@@ -298,12 +300,19 @@ public String getSchemaType(Schema p) {
 
         if (ref != null && !ref.isEmpty()) {
             type = openAPIType;
-        } else if (typeMapping.containsKey(openAPIType)) {
-            type = typeMapping.get(openAPIType);
-            if (languageSpecificPrimitives.contains(type))
-                return (type);
-        } else
-            type = openAPIType;
+        } else {
+            // Handle "any type" as an empty interface
+            if (openAPIType == "object" && p != null && !ModelUtils.isObjectSchema(p) && !ModelUtils.isMapSchema(p)) {
```

Comment:
@ashanbrown Please use `"object".equals(openAPIType)` instead

Reply:
@ashanbrown can you please fix it? Here in Arduino we really appreciate to have this PR merged :)

Reply:
Sure @eclipse1985, I've made that change.  It wasn't sure that the team was really offering to merge this though because I didn't see further comments from @antihax, @bvwells or  @grokify about this or my follow-up comment at https://github.com/OpenAPITools/openapi-generator/pull/888#issuecomment-415777718.  I'd still be happy to have this merged as is but my impression was there might be a larger issue here.